src/case-monitoring/supplier-utils.ts

```diff
@@ -221,6 +221,15 @@ export class ProcessExecutor {
         });
       logProcessExecution('DONE call execution of next step', executionStep.nextExecutionStep);
     } else if (executionStep.isLastStep) {
+      // Display notification
+      const notyf = configureToast(3000);
+      // Event_13tn0ty is the abort event
+      if (executionStep.id === 'Event_13tn0ty') {
+        toast(notyf, 'The email is aborted');
```

Comment:
**nitpick:** In #47 I suggested to use a different color in case of abort to distinguish between the case when the mail is sent and when it is not.
Is there a technical problem that prevents this solution from being implemented or is it just an oversight or  or misunderstanding?

Reply:
An oversight, will fix it ;)

Reply:
fixed in [da7054a](https://github.com/process-analytics/bonita-day-demo-2023/pull/94/commits/da7054a10a77b58cb54b610f934571e20b271603). We may need to adjust the colors and the notification text of [a777248](https://github.com/process-analytics/bonita-day-demo-2023/pull/94/commits/a777248f2777421e918611a5af439e2152a7e0b5).